frontend/src/components/service/Details.tsx

```diff
@@ -70,6 +71,13 @@ export default function ServiceDetails() {
                         <ValueLabel>{port}</ValueLabel>
                         <InlineIcon icon="mdi:chevron-right" />
                         <ValueLabel>{targetPort}</ValueLabel>
+                        <PortForward
+                          containerPort={targetPort}
+                          service={item}
+                          isPod={false}
```

Comment:
I guess `isPod` is the same as `!service`, so I don't think we need this property (makes things simpler).
Probably the same can be done for name and namespace. The more logic we abstract in the component, the less we have to worry about maintaining how it is called.